stylesheets/scss/print.scss

```diff
@@ -1,10 +1,81 @@
 /*
 	Print stylesheet.
-	
+
 	Include the variable and mixin files, and others as neccesary.
-	
+
+	Add anything you want to hide near the end of the file
+	along with the screen reader elements and skip links.
+
 */
 
 @import 'includes/fontawesome';
 @import 'variables';
 @import 'mixins';
+
+@media print {
+    *,
+    *:before,
+    *:after {
+        background: transparent !important;
+        color: #000 !important; // Black prints faster
+        box-shadow: none !important;
+        text-shadow: none !important;
+    }
+
+    a,
+    a:visited {
+        text-decoration: underline;
+    }
+
+    a[href]:after {
+        content: " (" attr(href) ")"; // this prints the url after the link text
+    }
+
+    abbr[title]:after {
+        content: " (" attr(title) ")";
+    }
+
+    // Don't show links for images, or javascript/internal links
+
+    a[href^="javascript:"]:after,
```

Comment:
I'd get rid of this line as to not suggest we endorse any kinds of shenanigans that we don't 😜